ci/mkpipeline.py

```diff
@@ -205,6 +205,11 @@ def visit(step: dict[str, Any]) -> None:
             if step.get("sanitizer") == "only":
                 step["skip"] = True
 
+            # Skip the Cargo driven builds if the Sanitizers aren't specified since everything
+            # relies on the Bazel builds.
+            if step.get("id") in ("rust-build-x86_64", "rust-build-aarch64"):
```

Comment:
I will add my steps in https://github.com/MaterializeInc/materialize/pull/29961 to this list, right?

Reply:
Why? I don't think we should skip the upload on non-Sanitizer builds.